astroscrappy/astroscrappy.pyx

```diff
@@ -1,6 +1,9 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-# cython: boundscheck=False, nonecheck=False, wraparound=False
-# cython: language_level=3, cdivision=True
+# cython: profile=True, boundscheck=False, nonecheck=False, wraparound=False
+# cython: cdivision=True, cython: linetrace=True
+# distutils: define_macros=CYTHON_TRACE=1
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
```

Comment:
I removed `profile`, `linetrace` and `CYTHON_TRACE` on master, since they are now added only when needed in the `setup.py` : 

https://github.com/astropy/astroscrappy/blob/master/setup.py#L83-L87

Reply:
Ok. That was an oversight on my part. I started on @jehturner's branch but then tried to cherry pick my changes so I must have missed that.